pkgs/games/angband/default.nix

```
@@ -1,4 +1,8 @@
-{ lib, stdenv, fetchFromGitHub, autoreconfHook, ncurses5 }:
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, ncurses5, SDL2, SDL2_image
+, SDL2_sound, SDL2_mixer, SDL2_ttf,
+# Override this to enable the SDL2 frontend for Angband, for instance:
+# pkgs.angband.override { enableSdl2 = true; }.
+enableSdl2 ? false }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, autoreconfHook, ncurses5
, enableSdl2 ? false, , SDL2, SDL2_image, SDL2_sound, SDL2_mixer, SDL2_ttf
}:
```

We can group it like this it is clear that those dependencies are for that option. We also can remove the comment because that is standard overlays which are documented in the docs.

---

Sure, that makes sense! I didn't see any info on formatting/formatters used. Is there no such thing in use?